graph/Graph/Graph.cpp

```diff
@@ -0,0 +1,93 @@
+//
+// Created by emyrmak on 7/9/2019.
+//
+#include <string>
+#include <iostream>
+#include "Graph.h"
+
+Graph::Graph(int& size_t, std::vector<std::vector<int>>& graph) : size(size_t) {
+    matrix = new int*[size];
+    visited = new bool[size];
+    distance = new int[size];
+    way = new std::string[size];
```

Comment:
We'd rather use just std::string, no pointer to it. 
std::string allocates memory automatically (when needed)